cdap-app-fabric/src/main/java/io/cdap/cdap/gateway/handlers/AppLifecycleHttpHandler.java

```diff
@@ -258,11 +258,12 @@ public void getAllApps(HttpRequest request, HttpResponder responder,
                          @QueryParam("pageToken") String pageToken,
                          @QueryParam("pageSize") Integer pageSize,
                          @QueryParam("orderBy") SortOrder orderBy,
-                         @QueryParam("nameFilter") String nameFilter
+                         @QueryParam("nameFilter") String nameFilter,
+                         @QueryParam("latestOnly") Boolean latestOnly,
+                         @QueryParam("strictEqual") Boolean strictEqual
```

Comment:
It's better to have something like `nameFilterType`. This would (a) show which field does it apply to as in future we may want to filter by more fields (b) allow more filter types but "containsIgnoreCase" and "equalsIgnoreCase".

Reply:
fixed